CHANGELOG.md

```diff
@@ -62,6 +62,7 @@
 * [FEATURE] Added `-<prefix>.s3.storage-class` flag to configure the S3 storage class for objects written to S3 buckets. #4300
 * [FEATURE] Add `freebsd` to the target OS when generating binaries for a Mimir release. #4654
 * [FEATURE] Ingester: Add `prepare-shutdown` endpoint which can be used as part of Kubernetes scale down automations. #4718
+* [ENHANCEMENT] Update the `rollout-operator` subchart to `0.4.2`. #4780
```

Comment:
```suggestion
```
The entry in `operations/helm/charts/mimir-distributed/CHANGELOG.md` is all that is needed since this change is only relevant to Helm. This might be confusing because there is an entry in this file elsewhere for rollout-operator `v0.4.0`, but that's referring to the image tag used in Jsonnet while this change only updates the helm chart.